lib/hermetic-common/src/schema.js

```diff
@@ -156,6 +161,52 @@ const technologySchema = Joi.object({
   authorisationNotes: Joi.string().optional().max(1024),
   externalIdentifiers: Joi.array().items(Joi.string()).optional()
     .description('A list of references to identifiers in external systems, e.g. SKUs'),
+  userbase: Joi.string().optional().valid('up to 10', '11 - 50', '51 - 500', '501 - 5000', 'over 5000')
+    .description(`Identify rough size of the userbase - options:
+up to 10, 11 - 50, 51-500, 501-5000, over 5000
+This is intended to give a rough idea of the system "size" through user numbers. `),
+  systemType: Joi.string().optional().max(100)
+    .description('Very short generic name which describes the essential purpose of the system e.g.Student Management System, Research Information Management System'),
+  commsResponsibility: Joi.string().optional().max(100)
+    .description('Where does responsibility for communications concerning system and process change reside? e.g. ITS, department (which one)'),
+  integrationResponsibility: Joi.string().optional().max(100)
+    .description('Who develops the code for system integrations e.g.ITS, vendor, third party?'),
```

Comment:
Not sure where discussions around some of these being lookup fields got to - it seems like it would like to be a lookup field

Reply:
After talking with Jeremy, he suggests using text fields for now, if it doesn't work then we will re look at it later.